kong/assets/service_checks.json

```diff
@@ -12,5 +12,38 @@
         ],
         "name": "Can Connect",
         "description": "Returns `CRITICAL` if the Agent is unable to connect to the Kong instance. Returns `OK` otherwise."
+    },
+    {
+        "agent_version": "7.29.0",
+        "integration": "Kong",
+        "check": "kong.datastore.reachable",
+        "statuses": [
+            "ok",
+            "critical"
+        ],
+        "groups": [
+            "host",
+            "endpoint"
+        ],
+        "name": "Datastore reachable",
+        "description": "Returns `CRITICAL` if Kong is unable to connect to the datastore. Returns `OK` otherwise."
+    },
+    {
+        "agent_version": "7.29.0",
+        "integration": "Kong",
+        "check": "kong.upstream.target.health",
+        "statuses": [
+            "ok",
+            "critical"
+        ],
+        "groups": [
+            "host",
+            "endpoint",
+            "address",
+            "target",
+            "upstream"
+        ],
+        "name": "Upstream target health status",
+        "description": "Returns `CRITICAL` if the target is unhealthy. Returns `OK` otherwise."
     }
```

Comment:
We should document openmetrics service check too

Reply:
Good catch ty